lfs/client.go

```diff
@@ -795,3 +820,12 @@ func setErrorHeaderContext(err error, prefix string, head http.Header) {
 		}
 	}
 }
+
+func PrintError(err error, format string, args ...interface{}) {
+	line := format
+	if len(args) > 0 {
+		line = fmt.Sprintf(format, args...)
+	}
+	fmt.Fprintln(ErrorWriter, line)
+	fmt.Fprintln(ErrorWriter, err.Error())
+}
```

Comment:
We have a `tracerx` package that logs stuff if you have pass `GIT_TRACE=1` to git lfs. This actually piggy backs on a core git feature, so running with git tracing on will show you all kinds of stuff :)